.github/workflows/ci.yml

```diff
@@ -25,20 +25,20 @@ jobs:
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@v1
     with:
       envs: |
-        - linux: test-oldestdeps-cov-xdist
+        - linux: test-py39-oldestdeps-cov
           python-version: 3.9
```

Comment:
This can be dropped now, open astronomy will set the python version from the toxenv's name if it includes the python version in it.